chromadb/test/property/strategies.py

```diff
@@ -31,12 +33,17 @@
 
 
 class EmbeddingSet(TypedDict):
+    """
+    An Embedding Set is a generated set of embeddings, ids, metadatas, and documents
+     that represent what a user would pass to the API.
+    """
+
     ids: types.IDs
     embeddings: Optional[types.Embeddings]
 
     # TODO: We should be able to handle None values
-    metadatas: Optional[Sequence[types.Metadata]]
-    documents: Optional[Sequence[types.Document]]
+    metadatas: Optional[List[types.Metadata]]
```

Comment:
The API expects List so we should type this the same way since the type checker doesn't allow List to conform to sequence for some reason